crates/turborepo-ui/src/tui/popup.rs

```diff
@@ -0,0 +1,60 @@
+use ratatui::{
+    layout::{Constraint, Flex, Layout, Rect},
+    text::Line,
+    widgets::{Block, List, ListItem, Padding},
+};
+
+use super::size::SizeInfo;
+
+const BIND_LIST_ITEMS: [&str; 11] = [
```

Comment:
issue: can we find some way to derive this information from the data where the keybindings are defined?  Perhaps that means adding an "info" field, for example.

Reply:
That's one of my TODOs. 😄 But I'm hoping so. These all get defined [here](https://github.com/vercel/turborepo/blob/e901df637d9fdee3a91907a2213d7f8cc01a45ef/crates/turborepo-ui/src/tui/input.rs#L54) and I wouldn't know how to translate that into a map I can use for these purposes.

Reply:
Chris confirmed for me [here](https://github.com/vercel/turborepo/pull/9545#issuecomment-2524306714) that it's worth a separate PR.